app/controllers/taxonomy_signups_controller.rb

```diff
@@ -0,0 +1,43 @@
+class TaxonomySignupsController < ApplicationController
+  def new
+    redirect_to '/' and return unless params[:paths].present?
+
+    taxon_path = params[:paths].first
+    load_taxon(taxon_path)
+    load_breadcrumbs
+  end
+
+  def confirm
+    redirect_to '/' and return unless params[:'taxon-list'].present?
+
+    taxon_path = params[:'taxon-list']
```

Comment:
Updated - I've opted to call the query param 'topic' across the controller and view. This is how we describe taxons to users so probably sensible to use that in the URL.